### ISSUE TITLE
Bitmapdata.Clone() program

### ISSUE THREAD

**Issue by zhongguo168a** [bp74/StageXL#223]
```
BitmapData clone([num pixelRatio]) {
    if (pixelRatio == null) pixelRatio = renderTextureQuad.pixelRatio;
    var bitmapData = new BitmapData(width, height, Color.Transparent, pixelRatio);
    bitmapData.drawPixels(this, this.rectangle, new Point<int>(0, 0));
    return bitmapData;
  }

factory BitmapData(int width, int height, [int fillColor = 0xFFFFFFFF, num pixelRatio = 1.0]);
```

In this code, width and height is num, but factory is int; So in dvm, it show error:

> Uncaught Unhandled exception:
> type 'double' is not a subtype of type 'int' of 'width'.
> #0      BitmapData.BitmapData (package:stagexl/src/display/bitmap_data.dart:34:26)
> #1      BitmapData.clone (package:stagexl/src/display/bitmap_data.dart:96:26)

**Comment by bp74** [comment]
Thanks for the bug report, i think this is related to #211

**Comment by bp74** [comment]
Fixed in 7c351090ee2eb56b862ced26efb0901525b93fe3
